packages/qdrant-loader-mcp-server/src/qdrant_loader_mcp_server/search/hybrid/adapters.py

```
@@ -13,6 +13,25 @@ def __init__(self, service: VectorSearchService):
     async def search(self, query: str, limit: int, project_ids: list[str] | None):  # type: ignore[override]
         return await self._service.vector_search(query, limit, project_ids)
 
+    def used_qdrant_hybrid_last_query(self) -> bool:
+        getter = getattr(self._service, "used_qdrant_hybrid_last_query", None)
+        if callable(getter):
+            try:
+                return bool(getter())
+            except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)